frame/claims/src/lib.rs

```diff
@@ -7,7 +7,27 @@ mod address;
 #[cfg(feature = "std")]
 pub use address::*;
 
+#[cfg(feature = "init-supply")]
+mod migration {
+	// --- substrate ---
+	use frame_support::migration::*;
+	// --- darwinia ---
+	use crate::*;
+
+	pub fn migrate<T: Trait>() {
+		sp_runtime::print("Migrating DarwiniaClaims...");
+
+		if let Some(_) = take_storage_value::<RingBalance<T>>(b"DarwiniaClaims", b"Total", &[]) {
```

Comment:
Substrate's StorageVersion control practice can been seen as a good practice, though we may do not need to use it in the case here:

https://github.com/paritytech/substrate/commit/497e8a094129fefa052ffb43918de8de8fe66c29#diff-32ce5961608780655bf3e00b2cc07d59L1252

Reply:
Yes, I've already review that. But the staking module is so complex.

This situation is similar to our.

https://github.com/paritytech/substrate/pull/5010